lib/log_parser.rb

```diff
@@ -19,31 +17,48 @@ def first_line
 
   def gamelog_json
     @gamelog_obj_ruby = {
-      @file_name => {
-        lines: line_count(),
-        players: [],
+      @file_name.to_sym => {
+        lines: @data.size(),
```

Comment:
Neste caso vc está chamando uma função e ela n precisa de nenhum parametro, entao aqui vc nao precisa de parenteses
```suggestion
        lines: @data.size,
```